packages/runtime/src/templates/vendor.ts

```diff
@@ -0,0 +1,19 @@
+// this file isn't meant to be imported.
+// it's a list of all external modules that we use,
+// and we vendor those into the `vendor/` directory
+// for consumption in other files.
```

Comment:
Just for future folks so they don't have to figure this out on their own - just short manual / explainer on how this is setup / how to interact:
```suggestion
// for consumption in other files.
// Vendoring happens automatically as part of runtime `build` script.
// You can trigger just vendoring without full package build by running `build:vendor` script.
```